search.php

```diff
@@ -14,30 +14,32 @@
 /**
  * Planet4 - Search functionality.
  */
-if ( is_main_query() && is_search() ) {
-	if ( 'GET' === filter_input( INPUT_SERVER, 'REQUEST_METHOD' ) ) {
-		$selected_sort    = filter_input( INPUT_GET, 'orderby', FILTER_SANITIZE_STRING );
-		$selected_filters = $_GET['f'] ?? ''; // phpcs:ignore
-		$filters          = [];
 
-		// Handle submitted filter options.
-		if ( $selected_filters && is_array( $selected_filters ) ) {
-			foreach ( $selected_filters as $type_name => $filter_type ) {
-				if ( ! is_array( $filter_type ) ) {
-					continue;
-				}
-				foreach ( $filter_type as $name => $filter_id ) {
-					$filters[ $type_name ][] = [
-						'id'   => $filter_id,
-						'name' => $name,
-					];
-				}
-			}
-		}
+// Limit access to cacheable method, to avoid spamming issues.
```

Comment:
Well in case of our search `GET` is equally uncacheable as the other verbs. Cloudflare doesn't cache stuff with a query string (with a few exceptions, but definitely not here). It also doesn't seem feasible to put a cache in front of ElasticSearch, given the large amount of possible URLs and requirements of fresh data.

Reply:
I don't mind checking for GET, though.

Reply:
Yep sorry I thought nginx was at least doing some caching there but there's only cache bypass when I try it